c2corg_ui/static/js/auth/authservice.js

```diff
@@ -71,6 +71,30 @@ app.Authentication.prototype.isAuthenticated = function() {
   return !!this.userData && !this.isExpired_();
 };
 
+/**
+ * @return {number}
+ * @private
+ */
+app.Authentication.prototype.getUserId_ = function() {
```

Comment:
At the moment it is used only once, I don't think it is worth having it: the least the best.